src/lib/Form/DataTransformer/SectionsTransformer.php

```diff
@@ -27,19 +29,48 @@ public function __construct(SectionService $sectionService)
         $this->sectionService = $sectionService;
     }
 
+    /**
+     * Transforms a domain specific Section objects into a string with comma separated Sections identifiers.
+     *
+     * @param mixed $value
+     *
+     * @return array|string
```

Comment:
Why does this return array or string? Feels more right to return a string or an empty string.